### PR TITLE
Fix the failing unet3d test.

### PR DESCRIPTION
**Pull request by vanbasten23**
Currently, it is failing with missing dataset. This PR is intended to provide the missing dataset.

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -44,6 +44,8 @@ export TEST_NAME=tf-nightly-mnist-func-v2-8
 jsonnet tests/oneshot.jsonnet -J . -S --tla-str test=$TEST_NAME | kubectl create -f -
 ```
 
+The above command will generate a job id such as `job.batch/pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`. To find the detail of the test, search in [GoogleCloud->Kubernetes->workload in the project `xl-ml-test`](https://pantheon.corp.google.com/kubernetes/workload/overview?mods=allow_workbench_image_override&project=xl-ml-test) with the job id `pt-nightly-unet3d-conv-v3-8-1vm-gz8ww`.
```
Comment (yeounoh) [inline tests/README.md:47]: I think this is helpful, thank you.

**Review by yeounoh — APPROVED** [review]
LGTM

File: tests/README.md (hunk 1 above)
Comment (vanbasten23) [reply tests/README.md:47]: Thanks!

**Comment by yeounoh** [comment]
Adding @will-cromar for the maintainer's review.

**Review by will-cromar — APPROVED** [review]

**Review by miladm — APPROVED** [review]
Thanks!

**Pull request merged**